app/controllers/fhir/r4/api_controller.rb

```diff
@@ -128,7 +128,7 @@ def update
       # so that they occur atomically
       ActiveRecord::Base.transaction do
         # Verify that the updated jurisdiction and other updates are valid
-        unless jurisdiction_valid_for_update?(patient) && patient.save(context: :api)
+        unless jurisdiction_valid_for_update?(patient) && patient.save(context: :api) && !fhir_map.values.any? { |v| v[:errors].present? }
```

Comment:
This should be changed back either to what it was originally:
```ruby
fhir_map.all? { |_k, v| v[:errors].blank? }
```
or
```ruby
fhir_map.none? { |_k, v| v[:errors].present? }
```
The reason given for changing to `any?` was for short-circuit evaluation, but both of these options will also short circuit and are more readable (and same for the other instances in this file).

Reply:
Ah yeah, reverted in ac815a8.